fleet_vehicle_inspection/views/fleet_vehicle_inspection_line.xml

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 - 2024, Marcel Savegnago - Escodoo https://www.escodoo.com.br
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="fleet_vehicle_inspection_line_form_view" model="ir.ui.view">
+        <field
+            name="name"
+        >fleet.vehicle.inspection.line.form (in fleet_vehicle_inspection)</field>
+        <field name="model">fleet.vehicle.inspection.line</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
+                        name="action_item_success"
+                        states="draft"
+                        string="Approve"
+                        type="object"
+                        class="oe_highlight btn-success"
+                        groups="fleet.fleet_group_manager"
+                    />
+                    <button
+                        name="action_item_failure"
+                        states="draft"
+                        string="Refuse"
+                        type="object"
+                        class="oe_highlight btn-danger"
+                        groups="fleet.fleet_group_manager"
+                    />
+                </header>
+                <sheet>
+                    <field name="result" invisible="1" />
+                    <div
+                        class="badge-pill badge-primary float-right"
```

Comment:
```suggestion
                        class="badge rounded-pill bg-primary float-end"
```

https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade_160.py

Also if you need to change the colour of the text you can add the class text-* (Ex: text-white, text-primary..)

Reply:
Thanks!